app/models/metadatum.rb

```diff
@@ -235,6 +235,21 @@ def validated_value
     ""
   end
 
+  def self.validated_value_multiget(metadata)
+    metadata_fields = MetadataField.where(id: metadata.pluck(:metadata_field_id)).index_by(&:id)
```

Comment:
you could also do .map(&:metadata_field) inside

Reply:
You mean like `metadata_fields = metadata.map(&:metadata_field).index_by(&:id)`? That would incur a query for each metadata record.
![image](https://user-images.githubusercontent.com/16944816/53528677-2cf55f00-3a9f-11e9-99fe-ade3d3817071.png)
![image](https://user-images.githubusercontent.com/16944816/53528693-354d9a00-3a9f-11e9-852d-19fcfddb2bb5.png)

Reply:
Ah ok, never mind then

Reply:
cool